src/doc/rustdoc/src/write-documentation/linking-to-items-by-name.md

```diff
@@ -103,6 +103,27 @@ macro_rules! foo {
 }
 ```
 
+The following suffixes are available:
+
+ * `!()` or `!{}` or `![]` for macros
+ * `()` for functions or methods
+
+And the list of disambiguators is as follows:
```

Comment:
I would do these as a comma separated list in the section where we teach prefixes instead of a separate thing. Something like:

These prefixes will be stripped when displayed in the documentation, so `[struct@Foo]` will be
rendered as `Foo`. The following prefixes are available: `struct`, `enum`, ........